receiver/oracledbreceiver/documentation.md

```diff
@@ -0,0 +1,54 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# oracledbreceiver
+
+## Metrics
+
+These are the metrics available for this scraper.
+
+| Name | Description | Unit | Type | Attributes |
+| ---- | ----------- | ---- | ---- | ---------- |
+| **oracledb.cpu_time** | Cumulative CPU time, in seconds | s | Sum(Double) | <ul> </ul> |
+| **oracledb.enqueue_deadlocks** | Total number of deadlocks between table or row locks in different sessions. | {deadlocks} | Sum(Int) | <ul> </ul> |
+| **oracledb.enqueue_locks.limit** | Maximum limit of active enqueue locks. | {locks} | Gauge(Int) | <ul> </ul> |
+| **oracledb.enqueue_locks.usage** | Current count of active enqueue locks. | {locks} | Gauge(Int) | <ul> </ul> |
+| **oracledb.enqueue_resources.limit** | Maximum limit of active enqueue resources. | {resources} | Gauge(Int) | <ul> </ul> |
+| **oracledb.enqueue_resources.usage** | Current count of active enqueue resources. | {resources} | Gauge(Int) | <ul> </ul> |
+| **oracledb.exchange_deadlocks** | Number of times that a process detected a potential deadlock when exchanging two buffers and raised an internal, restartable error. Index scans are the only operations that perform exchanges. | {deadlocks} | Sum(Int) | <ul> </ul> |
+| **oracledb.execute_count** | Total number of calls (user and recursive) that executed SQL statements | {executions} | Sum(Int) | <ul> </ul> |
+| **oracledb.hard_parses** | Number of hard parses | {parses} | Sum(Int) | <ul> </ul> |
+| **oracledb.logical_reads** | Number of logical reads | {reads} | Sum(Int) | <ul> </ul> |
+| **oracledb.parse_count_total** | Total number of parse calls. | {parses} | Sum(Int) | <ul> </ul> |
```

Comment:
`oracledb.execute_count` and `oracledb.parse_count_total` are not consistent with each other and with other metrics. Maybe `oracledb.executions` and `oracledb.parse_calls`?